src/main/java/org/folio/innreach/repository/MaterialTypeMappingRepository.java

```diff
@@ -16,4 +16,6 @@
 
   Optional<MaterialTypeMapping> findOneByCentralServerIdAndMaterialTypeId(UUID centralServerId, UUID materialTypeId);
 
+  Optional<MaterialTypeMapping> findOneByCentralServerIdAndCentralItemType(UUID centralServerId, Integer centralItemType);
```

Comment:
As I remember there might be several Folio material types mapped to the same central item type. Do you handle this situation properly?

Reply:
Was using wrong service for this mapping. Switched to ItemTypeMappingService